File: Pemrograman1/Modul3/C/PRAK305-2310817310009-Muhamamd_Fauzan_Ahsani.c
#include <stdio.h>
#include <math.h>

int main(){
    int sec, days, hours, minutes, seconds;
    scanf("%d", &sec);

    hours = floor(sec / 3600);
    minutes = floor(sec / 60);
    if (minutes >=60){
        minutes = minutes - floor(minutes / 60) * 60;
    }
    seconds = sec % 60;

    if (hours >= 24){
        days = floor(hours/24);
        hours = hours - days*24;
        printf("%d hari %02d:%02d:%02d", days, hours, minutes, seconds);
    }
    else {
        printf("%02d:%02d:%02d", hours, minutes, seconds);
    }
    return 0;
}

File: Pemrograman1/Modul3/Python/PRAK305-2310817310009-Muhamamd_Fauzan_Ahsani.py
sec = int(input())

hours = sec // 3600

minutes = sec // 60
if minutes >= 60:
    minutes = minutes - (minutes//60 * 60)

seconds = sec % 60

if hours >= 24:
    days = hours // 24
    hours = hours - days * 24
    print(f"{days} hari {hours:02d}:{minutes:02d}:{seconds:02d}")
else:
    print(f"{hours:02d}:{minutes:02d}:{seconds:02d}")